.github/workflows/jekyll.yml

```diff
@@ -1,9 +1,10 @@
 name: Build and deploy Jekyll site to GitHub Pages
 
 on:
-  push:
-    branches:
-      - main
+  workflow_dispatch:
+  release:
+    types:
+      - published
```

Comment:
If you want to do what @jhidding suggested in [this comment](https://github.com/esciencecenter-digital-skills/workshop-template/issues/121#issuecomment-1181975301), you could try:

```yaml
on:
  release:
    types: [created]
```
For example, I use it [here](https://github.com/sentinel-energy/friendly_data/blob/master/.github/workflows/python-publish.yml) to upload a new release to PyPI,